### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: fastlane/lib/fastlane/actions/clean_build_artifacts.rb [hunk 1]
```diff
@@ -35,12 +35,12 @@ def self.available_options
       end
 
       def self.description
-        "Deletes files created as result of running ipa, cert, sigh or download_dsyms"
+        "Deletes files created as result of running gym, cert, sigh or download_dsyms"
```
Comment (ohwutup) [inline fastlane/lib/fastlane/actions/clean_build_artifacts.rb]: should this be _gym_?

**Review by ohayon — CHANGES_REQUESTED** [review]
Just that one little change but otherwise 👍

File: fastlane/lib/fastlane/actions/clean_build_artifacts.rb (hunk 1 above)
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/clean_build_artifacts.rb]: Yes

Comment (ohwutup) [inline fastlane/lib/fastlane/actions/clean_build_artifacts.rb]: also should actions names maybe also be italicized?

Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/clean_build_artifacts.rb]: The description is printed in the action list also, so we don't always render it in markdown

Comment (ohwutup) [inline fastlane/lib/fastlane/actions/clean_build_artifacts.rb]: ohhh! got it!

**Review by ohayon — APPROVED** [review]

**Pull request merged**